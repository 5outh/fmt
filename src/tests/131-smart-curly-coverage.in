<?php
//passes:OrderUseClauses,AddMissingCurlyBraces,SmartLnAfterCurlyOpen,NormalizeLnAndLtrimLines,MergeParenCloseWithCurlyOpen,MergeCurlyCloseAndDoWhile,MergeDoubleArrowAndArray,ExtraCommaInArray,ResizeSpaces,Reindent,ReindentColonBlocks,ReindentIfColonBlocks,ReindentObjOps,EliminateDuplicatedEmptyLines,AlignEquals,AlignDoubleArrow,LeftAlignComment,RTrim


if ($a) { "$a";

}

if ($a) { "${a}";

}

if ($a) {{ "$a";

}
}

if ($a) {{ "${a}";

}
}
