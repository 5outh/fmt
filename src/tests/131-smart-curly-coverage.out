<?php
//passes:OrderUseClauses,AddMissingCurlyBraces,SmartLnAfterCurlyOpen,NormalizeLnAndLtrimLines,MergeParenCloseWithCurlyOpen,MergeCurlyCloseAndDoWhile,MergeDoubleArrowAndArray,ExtraCommaInArray,ResizeSpaces,Reindent,ReindentColonBlocks,ReindentLoopColonBlocks,ReindentIfColonBlocks,ReindentObjOps,EliminateDuplicatedEmptyLines,AlignEquals,AlignDoubleArrow,LeftAlignComment,RTrim

if ($a) {
	"$a";

}

if ($a) {
	"${a}";

}

if ($a) {
	{"$a";

	}
}

if ($a) {
	{"${a}";

	}
}
